api/api.go

```diff
@@ -89,6 +92,11 @@ var spaceIDRegexp = regexp.MustCompile("^[a-zA-Z0-9_]+$")
 
 type SpaceID string
 
+func NewSpaceID() SpaceID {
+	id := ksuid.New()
+	return SpaceID(fmt.Sprintf("sp_%s", id.String()))
+}
+
```

Comment:
I think this should live in ppl/zqd/apiserver since isn't really part of the API.